libcyphal_demo/src/platform/common_helpers.hpp

```diff
@@ -0,0 +1,166 @@
+// This software is distributed under the terms of the MIT License.
+// Copyright (C) OpenCyphal Development Team  <opencyphal.org>
```

Comment:
All these "platform/**" files are brought from libcyphal repo (from its "examples") almost as is. The main difference is that here I don't use any of c++ heap allocating containers (std::vector, std::string, etc).